### PR TITLE
minor change to argcomplete_command

### LINKED ISSUE #1440: Correction to setup.py
The current `setup.py` gives errors when installing `python-argcomplete`. 
image
-  ```bash
   usage: activate-global-python-argcomplete [-h] [--dest DEST] [--user]
                                          [--no-defaults]
                                          [--complete-arguments ...]
   activate-global-python-argcomplete: error: [Errno 13] Permission denied: 
   '/etc/bash_completion.d/python-argcomplete'
   Please try --user to install into a user directory, or --dest to specify the bash completion modules directory```

 This error, to my knowledge occurs only for `zsh`.  Remedy for this is to change:
 ` activate-global-python-argcomplete` ---> `activate-global-python-argcomplete --user`
 https://github.com/weecology/retriever/blob/be301cf9b0643fec8c26d67a3852c7d47f036d18/setup.py#L131

---
-    ```bash
      sh: 18: eval: Syntax error: "(" unexpected (expecting "}")```
   This can be corrected by replace `()` with `{}`  https://github.com/weecology/retriever/blob/be301cf9b0643fec8c26d67a3852c7d47f036d18/setup.py#L123

### PR DESCRIPTION
**Pull request by DumbMachine**
fixes #1440

### CODE REVIEW

**Comment by DumbMachine** [comment]
travis seems to fail because of:
image

File: setup.py [hunk 1]
```diff
@@ -120,15 +120,15 @@ def read(*names, **kwargs):
...
+    argcomplete_command = "eval $(register-python-argcomplete retriever)"
     with open(os.path.expanduser(bash_file), "a+") as bashrc:
         bashrc.seek(0)
         # register retriever for arg-completion if not already registered
         # whenever a new shell is spawned
         if argcomplete_command not in bashrc.read():
             bashrc.write(argcomplete_command + "\n")
             bashrc.close()
-    os.system("activate-global-python-argcomplete")
+    os.system("activate-global-python-argcomplete --user")
```
Comment (henrykironde) [inline setup.py]: is --user needed ?

Comment (DumbMachine) [reply setup.py]: Yes, now having that wil lead to :
```bash

Installing bash completion script /etc/bash_completion.d/python-argcomplete
usage: activate-global-python-argcomplete [-h] [--dest DEST] [--user]
                                          [--no-defaults]
                                          [--complete-arguments ...]
activate-global-python-argcomplete: error: [Errno 13] Permission denied: '/etc/bash_completion.d/python-argcomplete'
Please try --user to install into a user directory, or --dest to specify the bash completion modules directory

```

Comment (henrykironde) [reply setup.py]: What OS are you using?

Comment (DumbMachine) [reply setup.py]: linux

Comment (henrykironde) [reply setup.py]: I will test it with other OS, any one  else is welcome to help with the testing and report back

**Comment by henrykironde** [comment]
Thanks @DumbMachine, this was added f4ed32d59972f4f0dd5b11243ba0f6a575b4a1d7 and we appreciate that you initiated this work. In case we happed to add the `--user` part, I will open this

**Pull request closed without merging**